### PR TITLE
new parser for rhv_log_collector_analyzer

### PR DESCRIPTION
**Pull request by dougsland**
Parser for rhv-log-collector-analyzer tool

### CODE REVIEW

File: insights/archive/repository/base_archives/rhel7/insights_commands/rhv-log-collector-analyzer_--json [hunk 1]
```diff
@@ -0,0 +1,188 @@
+{
```
Comment (bfahr) [inline insights/archive/repository/base_archives/rhel7/insights_commands/rhv-log-collector-analyzer_--json]: @dougsland it is not necessary to add this to the archive template.  It is used to test the archive code.

File: insights/parsers/rhv_log_collector_analyzer.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+"""
+RHV Log Collector Analyzer
+=================================
+
```
Comment (bfahr) [inline insights/parsers/rhv_log_collector_analyzer.py]: In order to include this parser in the documentation, please add a new file `docs/shared_parsers_catalog/rhv_log_collector_analyzer.rst` which looks like this:
```
.. automodule:: insights.parsers.rhv_log_collector_analyzer
   :members:
   :show-inheritance:
```
You can find information on writing and testing documentation here: https://insights-core.readthedocs.io/en/latest/docs_guidelines.html

I believe when you attempt to build the docs you will get an error from Sphinx caused by the `===...` on line 3 being longer than the title string above it.  They must be the same length.

File: insights/archive/repository/base_archives/rhel7/insights_commands/rhv-log-collector-analyzer_--json (hunk 1 above)
Comment (dougsland) [reply insights/archive/repository/base_archives/rhel7/insights_commands/rhv-log-collector-analyzer_--json]: Thanks, removing.

File: insights/parsers/rhv_log_collector_analyzer.py (hunk 2 above)
Comment (dougsland) [reply insights/parsers/rhv_log_collector_analyzer.py]: good catch, done

**Pull request merged**